src/comm/Connection.h

```diff
@@ -149,6 +149,9 @@ class Connection : public RefCountable
     /** Netfilter MARK values currently sent on this connection */
     nfmark_t nfmark;
 
+    /** Netfilter CONNMARK value of this connection */
+    nfmark_t nfConnmark;
```

Comment:
missing initialization. Please use C++11 style "nfmark_t nfConnmark = 0;" here in the header, its fine and somewhat reduces the diff.